### PR TITLE
[ENDPOINT] DELETE /testimonials/:id - OT102-86

### PR DESCRIPTION
**Pull request by JonatanGarbuyo**
[OT102-86](https://alkemy-labs.atlassian.net/browse/OT102-86)
## Description:

AS admin user
I WANT to delete Testimonials
TO moderate Testimonials about the NGO.

## Acceptance criteria:

DELETE / testimonials /: id.

#### testimonials before delete
get-testimonials
db-before

#### testimonials after delete
db-after

### CODE REVIEW

**Review by dvelezroman — APPROVED** [review]
Excelente

File: services/testimonials.js [hunk 1]
```diff
@@ -27,3 +28,11 @@ exports.updateById = async (id, name, content) => {
     throw new ErrorObject(error.message)
   }
 }
+
+exports.deleteTestimonial = async (id) => {
+  try {
+    return await Testimonial.destroy({ where: { id } })
```
Comment (gabyMangoni) [inline services/testimonials.js:38]: podrias agregar un 
if (!testimonial) {
      throw Error(`ID ${id} already deleted.`)
    } else {

para unificar criterios...

Comment (JonatanGarbuyo) [reply services/testimonials.js:38]: Listo

**Review by gabyMangoni — APPROVED** [review]
All Ok

**Review by joselo85 — APPROVED** [review]

**Pull request merged**